pkg/controller/cloud/node_controller.go

```diff
@@ -273,7 +273,7 @@ func (cnc *CloudNodeController) MonitorNode() {
 				// doesn't, delete the node immediately.
 				exists, err := ensureNodeExistsByProviderID(instances, node)
 				if err != nil {
-					glog.Errorf("Error getting data for node %s from cloud: %v", node.Name, err)
+					glog.Errorf("Error checking if node %s is shutdown: %v", node.Name, err)
```

Comment:
This should be 
```
Error checking if node %s exists: %v
```

Reply:
error return should be in lower case, like this?
```glog.Errorf("error checking if ...")```

Reply:
Generally I agree but we don't follow that in a bunch of other places in this file so maybe something to do as a follow up :)